docs/source/APIs/gen

```diff
@@ -31,7 +31,7 @@ def doFile(name, dir, ent):
         top = ast.parse(fp.read())
 
     ary = []
-    base = ent[ : -3]
+    base = ent[:-3]
```

Comment:
I always disagree with black on this one.  Colon is a binary operator.  There should be spaces around it.  And also, I didn't think this file was under auto/enforced formatting.  GitHub doesn't seem to think it is.

Reply:
pre-commit does this:
```
        poetry run pre-commit run --all-files
```